frontend/public/index.html

```diff
@@ -3,7 +3,33 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title>Technigo React App</title>
+    <title>Tika and Maria's Project Authentication</title>
+
+    <!-- Meta tags for facebook -->
```

Comment:
Great job with meta-tags! We wish you could give us a crash-course in this topic 😉